website/client/src/pages/settings/siteDataRows/userDataRow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <fragment>
+    <tr
+      v-if="!modalVisible"
+    >
+      <td class="settings-label">
+        {{ $t("yourUserData") }}
+      </td>
+      <td class="settings-value">
+      </td>
+      <td class="settings-button">
+        <a
+          class="edit-link"
+          @click.prevent="openModal()"
+        >
+          {{ $t('learnMore') }}
+        </a>
+      </td>
+    </tr>
+    <tr
+      v-if="modalVisible"
+      class="expanded"
+    >
+      <td colspan="3">
+        <div
+          v-once
+          class="dialog-title"
+        >
+          {{ $t("yourUserData") }}
+        </div>
+        <div
+          v-once
+          class="dialog-disclaimer"
+        >
+          {{ $t("yourUserDataDisclaimer") }}
+        </div>
+
+        <div class="d-flex justify-content-center data-download-selection">
+          <table>
+            <tr>
+              <td>Task History</td>
```

Comment:
These should be translated--we can get by without if they're developer tools only, but I think these are end user facing?

Reply:
replaced them all with the translated strings